blocks/library/embed/index.js

```diff
@@ -134,23 +134,14 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 				const { setAttributes, focus, setFocus } = this.props;
 				const updateAlignment = ( nextAlign ) => setAttributes( { align: nextAlign } );
 
-				const controls = [
-					focus && (
-						<BlockControls key="controls">
-							<BlockAlignmentToolbar
-								value={ align }
-								onChange={ updateAlignment }
-							/>
-						</BlockControls>
-					),
-					focus && (
-						<InspectorControls key="inspector">
-							<BlockDescription>
-								<p>{ __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ) }</p>
-							</BlockDescription>
-						</InspectorControls>
-					),
-				];
+				const controls = focus && (
+					<BlockControls key="controls">
```

Comment:
This `key` property isn't necessary anymore.

Reply:
Ame as above